components/scream/src/share/io/tests/io.cpp

```diff
@@ -122,6 +122,7 @@ TEST_CASE("input_output_basic","io")
   auto avg_params = get_in_params("Average",io_comm);
   auto min_params = get_in_params("Min",io_comm);
   auto max_params = get_in_params("Max",io_comm);
+  auto multi_params = get_in_params("Multisnap",io_comm);
```

Comment:
You should update the cocmment above. We no longer expect 4 files, but 5. 1 file each for average, instantaneous, min and max data, and multiple time snapshots.